clusterloader2/testing/experimental/storage/pod-startup/volume-types/persistentvolume/pod_with_pvc.yaml

```diff
@@ -0,0 +1,27 @@
+{{$index := .Index}}
+{{$appName := .AppName}}
+{{$volumesPerPod := .VolumesPerPod}}
+apiVersion: v1
+kind: Pod
```

Comment:
I think the same concern about [using raw volumes instead of a controller](https://github.com/kubernetes/perf-tests/issues/666) is valid for this one.

Reply:
can you wait on that PR until this one is merged?

Reply:
I agree - let's have this one merged and then migrate to deployments all in one shot.